eva/catalog/catalog_type.py

```diff
@@ -98,3 +98,20 @@ class IndexType(EVAEnum):
     @classmethod
     def is_faiss_index_type(cls, t):
         return t in [cls.HNSW]
+
+
+class ColumnName(EVAEnum):
```

Comment:
Like this! Maybe make the name more descriptive and add similar logic for images

Reply:
How about we call them VideoColumnNames?